src/battle_game/app/app.cpp

```diff
@@ -451,3 +451,4 @@ void App::UpdateImGui() {
   ImGui::Render();
 }
 }  // namespace battle_game
+// pull request test
```

Comment:
Unrealeted file changes like this (or something in `fonts` folder), should not be included to commit and push.